docs/001_develop/03_client-capabilities/008_interaction/file-upload.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: 'File Upload'
+sidebar_label: 'File Upload'
+id: client-interaction-file-upload
+keywords: [interaction, interactive, upload, file, data]
+tags:
+- interaction
+- interactive
+- upload
+- file
+- data
+sidebar_position: 8
+---
+import FileUploadDemo from '../../../../examples/ui/client-capabilities/interaction/fileupload.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# File Upload
+
+Use cases:
+* Uploading files to the server
+
+The file-upload component can be used to select single or multiple files from the local file system. Selected files can be uploaded to a server using the **Upload** button.
+
+## Example
+<FileUploadDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration
+```html
+  <rapid-file-upload
+    label="File Upload (JSON files only)"
+    accept="application/json"
+    file-size-limit-bytes="104857600"
+    uploaded-files-resource-name="ALL_FILE_ATTACHMENTS"
+    entity-id="Demo"
+    field-name="ENTITY_ID"
+    upload-key="series"
+  >
+  </rapid-file-upload>
```

Comment:
Nitpick, but the declerations in the other places have just had the tag with no attributes or properties set. Same for react and angular
```suggestion
<rapid-file-upload></rapid-file-upload>
```